pom.xml

```diff
@@ -357,6 +357,126 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>licenses</id>
+      <properties>
+        <openshiftio.dir>${project.basedir}/.openshiftio</openshiftio.dir>
+        <licenses.dir>${project.basedir}/licenses</licenses.dir>
+
+        <license-maven-plugin.version>1.13</license-maven-plugin.version>
+        <xml-maven-plugin.version>1.0.1</xml-maven-plugin.version>
+      </properties>
+
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-clean-plugin</artifactId>
+            <configuration>
+              <filesets>
+                <fileset>
+                  <directory>${licenses.dir}</directory>
```

Comment:
It is a temporary directory ? Should it be shipped ? 

In case of 1, it should be in `target`.
In case of 2, if should be in `src`.

If none, why do we delete it ?